cartographer_ros/cartographer_ros/pbstream_to_ros_map_main.cc

```diff
@@ -127,6 +146,15 @@ int main(int argc, char** argv) {
   CHECK(!FLAGS_pbstream_filename.empty()) << "-pbstream_filename is missing.";
   CHECK(!FLAGS_map_filestem.empty()) << "-map_filestem is missing.";
```

Comment:
It is not necessary to insist on these options if the user wants to publish the map.